ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -541,7 +536,7 @@ export const ConceptHomepage = fp.flow(
                               }
                               key={i}
                               data-test-id='domain-box'
-                              updating={domainsLoading.includes(domain.domain)}
+                              updating={loadingConceptCounts}
```

Comment:
We can remove the `updating` prop and the spinner from the `DomainCard` component. Since we're not making separate api calls for each card, we don't need to display a spinner

Reply:
We can do a follow up ticket for these if you prefer. Won't affect anything, just cleanup after the api change

Reply:
updating property is needed for the DomainCards to load, when we first come to the page.